changelog/@unreleased/pr-210.v2.yml

```diff
@@ -0,0 +1,6 @@
+type: improvement
+improvement:
+  description: Make balanced URI scoring optional behavior that can be enabled by
```

Comment:
Can you update this to reflect what consumers of this library would care about without them having to look at the code. For example, the `WithBalancedURIScoring` option can now be used to enable this feature, where the default. Based on other comments below, whatever we decide on the user facing option, we should highlight that here and mention that it differs from default